qx/handle_tx.go

```diff
@@ -117,22 +129,6 @@ func TxSign(privkeyStrs []string, rawTxStr string, network string, pks []string)
 	case "mixnet":
 		param = &params.MixNetParams
 	}
-	for _, privkeyStr := range privkeyStrs {
-		privkeyByte, err := hex.DecodeString(privkeyStr)
-		if err != nil {
-			return "", err
-		}
-		if len(privkeyByte) != 32 {
-			return "", fmt.Errorf("invaid ec private key bytes: %d", len(privkeyByte))
-		}
-		privateKey, pubKey := ecc.Secp256k1.PrivKeyFromBytes(privkeyByte)
-		h160 := hash.Hash160(pubKey.SerializeCompressed())
-		addr, err := address.NewPubKeyHashAddress(h160, param, ecc.ECDSA_Secp256k1)
-		if err != nil {
-			return "", err
-		}
-		privateKeyMaps[addr.String()] = privateKey
-	}
 	if len(rawTxStr)%2 != 0 {
```

Comment:
Odd length hexadecimal string is very common, we just need to correct it automatically:
`
if len(rawTxStr)%2 != 0 {
	rawTxStr = "0" + rawTxStr
}
`